zetasql.yaml

```diff
@@ -28,25 +30,41 @@ environment:
       - tzdata
       - wolfi-baselayout
   environment:
-    BAZEL_ARGS: "--config=g++"
-    EXTRA_BAZEL_ARGS: "--tool_java_runtime_version=local_jdk"
+    BAZEL_ARGS: "--config=g++ --cxxopt=-std=c++17 --host_cxxopt=-std=c++17 --tool_java_runtime_version=local_jdk"
     JAVA_HOME: /usr/lib/jvm/java-17-openjdk
```

Comment:
c++17 is old/low; it can cause issues with runtime compat with forexample our icu. But if that is not used, it can also be ok. Higher / default version standard would be nice (unless known to be broken).